CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 * Add AutoCorrect for StatementPositionRule.  
   [Raphael Randschau](https://github.com/nicolai86)
 
+* Add ConditionalBindingCascadeRule.
```

Comment:
This is missing the two trailing spaces as described in [CONTRIBUTING.md](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes).